src/DocumentDB.ChangeFeedProcessor/Monitoring/HealthEventLevel.cs

```diff
@@ -0,0 +1,22 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.Monitoring
+{
+    /// <summary>
+    /// The healthiness level
+    /// </summary>
+    public enum HealthEventLevel
```

Comment:
I think HealthIssueSeverity would be better name. What do you think? Also it would be good to have increasing values as severity decreases, so we need to have minimal values, others can be added later. For minimal I think we should have Critical and Error. If we need Informational (or Healthy, or something like that), we should also have Warning.